cluster/gce/gci/configure-helper.sh

```diff
@@ -2570,9 +2600,10 @@ function main() {
     fi
   fi
 
-  # generate the controller manager and scheduler tokens here since they are only used on the master.
+  # generate the controller manager, scheduler and CA tokens here since they are only used on the master.
```

Comment:
CA often refers to certificate authority. This abbreviation is confusing.

Reply:
Done.